packages/client-adapters/src/adapters/lexicons/adapter.js

```diff
@@ -147,13 +147,21 @@ class AlpheiosLexiconsAdapter extends BaseAdapter {
   * @param {String} url - url from what we need to cache data
   * @return {Boolean} - true - if cached is successed
   */
-  async checkCachedData (url) {
+  async checkCachedData (url, externalData = null, skipFetch = false) {

```

Comment:
please add the new parameters to the JSDOC

Reply:
Missed that

Reply:
Fixed